package.json

```diff
@@ -53,6 +53,7 @@
     "eslint-plugin-standard": "^3.0.1",
     "intelli-espower-loader": "^1.0.1",
     "mocha": "^4.0.1",
+    "mock-fs": "^4.4.2",
```

Comment:
I just took a look at this library and it is definitely going to be broken by Node.js master (v10)...it might worth adding it to citgm-all and see how we could fix it. I also had some conversation with @jasnell yesterday about providing a hook so testing libraries can monkey-patch this kind of thing reliably...

Reply:
BTW to mock the fs methods, simply overriding `fs.something` on the require cache should be enough

Reply:
Okay, I didn't know. Then, what we should do with this?

Reply:
@alopezsanchez I think simply overriding `fs.something = mock` in the hooks is enough?

Reply:
@joyeecheung, done!

But... I think I made a mistake. There was a conflict with `package.json` and I resolved it with GitHub and it obviously generated 5d3d0005867b30b20b9f5d215a59c66c5a6ab7e5.

This is not correct, right? How can I solve it without worsening the log?

Reply:
Hmm. Sorry for merge conflict's after npm packages update. In future you definitely want to handle this locally by pull latest master through rebase way and not using github's editor.

Any just pull the latest master by `git pull upstream master --rebase` and fix all the conflict and push it and this should be fixed. Also while you at it you should probably remove the last commit `Merge branch ...` and then do this.

Reply:
Hello @joyeecheung @cPhost. Sorry, I totally forgot this PR. I resolved the commit issue!